modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-checkpoint-connector.adoc

```diff
@@ -96,6 +96,10 @@ After the connector is created:
 * Ensure that there are no errors in logs and in Redpanda Console.
 * Wait for the Kafka cluster topics connector to catch up. Then check to confirm that the consumer groups are replicated.
 
+== Using the Connectors API
+When using Connectors API, instead of `source.cluster.sasl.username` and `source.cluster.sasl.password` configuration fields
+you can specify a value for the `source.cluster.sasl.jaas.config` field.
```

Comment:
```suggestion
== Use the Connectors API
When using the Connectors API, instead of specifying `source.cluster.sasl.username` and `source.cluster.sasl.password`, 
you can specify a value for `source.cluster.sasl.jaas.config`.
```